docker/oss/Dockerfile

```diff
@@ -64,9 +64,11 @@ ARG GRAYLOG_UID=1100
 ARG GRAYLOG_GROUP=graylog
 ARG GRAYLOG_GID=1100
 
+COPY wait-for-it.sh /wait-for-it.sh
```

Comment:
Do we need to document the usage of this script?

Reply:
Yeah, that'd be good. It's for docker-compose, to make it wait on Elasticsearch to be available before attempting to start Graylog. I ran into an intermittent issue where Graylog would fail to start because Elasticsearch was still booting up. The `depends_on` does not accomplish the task properly, but this script makes it work beautifully.